internal/client/gitlab.go

```diff
@@ -208,11 +208,11 @@ func (c *gitlabClient) CreateRelease(ctx *context.Context, body string) (release
 	name := title
 	tagName := ctx.Git.CurrentTag
 	release, resp, err := c.client.Releases.GetRelease(projectID, tagName)
-	if err != nil && (resp == nil || resp.StatusCode != 403) {
+	if err != nil && (resp.StatusCode != 403 && resp.StatusCode != 404) {
```

Comment:
why remove the `resp == nil` check?

Reply:
I removed it, because the `resp` is now `nil` when no release is present. So the error condition always triggered. If  we want to keep the `nil` check, we will have to change the conditional logic a bit more

Reply:
Never mind, my bad. I added back the `nil` check, definitely makes sense.